From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
Date: Sun, 18 Apr 2021 22:48:56 +0200
Subject: [ARCHEOLOGY] 150balbes fix station legacy (#2782)

> X-Git-Archeology: > recovered message: > * Fix Legacy Stations
> X-Git-Archeology: > recovered message: > Co-authored-by: balbes150 <balbes-150@yandex.ru>
> X-Git-Archeology: - Revision 510e348202f8efb838a57a9eec6ccbc910d7e06e: https://github.com/armbian/build/commit/510e348202f8efb838a57a9eec6ccbc910d7e06e
> X-Git-Archeology:   Date: Sun, 18 Apr 2021 22:48:56 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: 150balbes fix station legacy (#2782)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dts b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dts
new file mode 100644
index 000000000000..99f3a24cadda
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dts
@@ -0,0 +1,7 @@
+/dts-v1/;
+#include "rk3399-roc-pc.dtsi"
+
+/ {
+	model = "Firefly roc-rk3399-pc";
+	compatible = "firefly,roc-rk3399-pc", "rockchip,rk3399";
+};
-- 
Armbian

